lessons/misc-quick-reference/cs-string.md

```diff
@@ -49,50 +49,189 @@ var message = $"{name} is a {description}";
 var mathProblem = $"What do you get when you multiply six by 9: {6 * 9}"
 ```
 
----
-
 ### String length
 
----
+We can get the length of a string by using the length property.
+
+```C#
+var name = "Suncoast Developers Guild";
+
+var nameLength = name.Length;
+
+Console.WriteLine(nameLength);
+//Output: 25;
+```
 
 ### Iterating the characters in a string
 
----
+We can iterate over a string and assess each character with a foreach loop.
+
+```C#
+var name = "Mary";
+
+foreach (var character in name)
+{
+  Console.WriteLine(character);
+}
+
+//Output: 
+//M
+//a
+//r
+//y
+```
+
+Alternatively, we can use the string length to use a for loop and access each character by its index.
+
+```C#
+var name = "Mary";
+
+for (var i = 0; i < name.Length; i++)
```

Comment:
No single letter variables allowed. So use `index`

Reply:
Still a bad habit I am trying to break! Corrected.